pkg/components/controllers.go

```diff
@@ -162,7 +163,10 @@ func startIngressController(cfg *config.MicroshiftConfig, kubeconfigPath string)
 		return err
 	}
 
-	if err := assets.ApplyDeployments(apps, renderTemplate, renderParamsFromConfig(cfg, nil), kubeconfigPath); err != nil {
+	extraParams := assets.RenderParams{
```

Comment:
I guess that for a decoupled CNI-plugin we may need to call the CNI plugin (know it's a CNI plugin) and call it to re-create the deployment.

We could make that part of a plugin interface i.e. have a "restart" call to the plugin which would add creation timestamps to the objects that need re-creation in the event of necessity.

Hmmmmm

Reply:
This is one reason to have MicroShift run the plugin scripts, instead of relying solely on systemd. @fzdarsky